### ISSUE TITLE
honorspy-1.2.lua:197: attempt to compare number with nil honorspy\honorspy-1.2.lua:197

### ISSUE THREAD

**Issue by keslol** [kakysha/HonorSpy#34]
/hs show command fails with

```
5x honorspy\honorspy-1.2.lua:197: attempt to compare number with nil
honorspy\honorspy-1.2.lua:197: in function <honorspy\honorspy.lua:196>
[C]: in function `sort'
honorspy\honorspy-1.2.lua:196: in function `BuildStandingsTable'
honorspy\GUI.lua:31: in function `Show'
honorspy\GUI.lua:73: in function `Toggle'
honorspy\honorspy-1.2.lua:174: in function <honorspy\honorspy.lua:174>
(tail call): ?
...\AceConfig-3.0-3\AceConfigCmd-3.0\AceConfigCmd-3.0-14.lua:143: in function <...\AceConfig-3.0\AceConfigCmd-3.0\AceConfigCmd-3.0.lua:133>
...\AceConfig-3.0-3\AceConfigCmd-3.0\AceConfigCmd-3.0-14.lua:408: in function <...\AceConfig-3.0\AceConfigCmd-3.0\AceConfigCmd-3.0.lua:334>
(tail call): ?
...\AceConfig-3.0-3\AceConfigCmd-3.0\AceConfigCmd-3.0-14.lua:769: in function `HandleCommand'
...\AceConfig-3.0-3\AceConfigCmd-3.0\AceConfigCmd-3.0-14.lua:781: in function `?'
FrameXML\ChatFrame.lua:4609: in function `ChatEdit_ParseText'
FrameXML\ChatFrame.lua:4270: in function `ChatEdit_SendText'
FrameXML\ChatFrame.lua:4306: in function <FrameXML\ChatFrame.lua:4302>
[C]: in function `ChatEdit_OnEnterPressed'
[string "*:OnEnterPressed"]:1: in function <[string "*:OnEnterPressed"]:1>

Locals:
a = <table> {
 1 = "Moringotho"
 2 = "ROGUE"
 4 = 0
 7 = 0
 8 = 1573904924
}
b = <table> {
 1 = "Sharana"
 2 = "PRIEST"
 3 = 2971
 4 = 0
 5 = 0
 6 = 0
 7 = 0
 8 = 1573909661
}
(*temporary) = nil
(*temporary) = 2971
(*temporary) = "attempt to compare number with nil"
sort_column = 3

```

**Comment by kakysha** [comment]
try purging your data (Esc -> Interface Options -> Addons -> HonorSpy -> _purge all data)

**Comment by keslol** [comment]
same problem, same playernames in the error message . other players in the guild can't access hs too

**Comment by kakysha** [comment]
fixed in v1.3.1, pls update
